sphinxcontrib/phpdomain.py

```diff
@@ -22,7 +22,7 @@
 from sphinx.util.nodes import make_refnode
 from sphinx.util.compat import Directive
 from sphinx.util.docfields import Field, GroupedField, TypedField
-
+from sphinx.domains.python import _pseudo_parse_arglist
```

Comment:
Your saying is right. I agree with you.

I considered it could use beyond module, because function is used both python and javascript domain.

I think our choices are:

1. Copy, not import, and use this function
2. Make a new better parser
3. other choices???

I can do 1st choice quickly, but as for 2nd, I don't know how long time I will take.